docs/metallb.md

```diff
@@ -0,0 +1,81 @@
+# MetalLB
```

Comment:
It is nice to have metallb.md under docs/ like this PR.
Now Kubespray repo contains README.md under roles/kubernetes-apps/metallb.
Please take a look at the file, merge the content into this metallb.md and remove the old one to avoid confusion later.

Reply:
I copied and re-worded the information from `roles/kubernetes-apps/metallb/README.md` when writing `docs/metallb.md`, I just moved a bit the lines around and removed the old file in the last push.